templates/monitoring/kube_state_metrics_3scale_alerts.yaml

```diff
@@ -94,15 +94,24 @@ spec:
         for: 5m
         labels:
           severity: critical
-      - alert: ThreeScalePodCount
+      - alert: ThreeScaleZyncPodAvailability
         annotations:
+          message:  3Scale Zync has {{ "{{" }} printf "%.0f" $value {{ "}}" }} in a ready state. Expected at least 2 pod.
```

Comment:
Should `Expected at least 2 pod` be `Expected at least 2 pods` (plural)? 

Or to be more descriptive, `Expected a minimum of 2 pods`. Just a suggestion.

Reply:
:+1: